Dockerfile

```diff
@@ -1,19 +1,17 @@
-FROM maven:3.5-jdk-7
-# Grab JDK from jessie-backports since default sources do not include openJDK-8
+FROM maven:3.6.1
 RUN set -ex && \
-    echo 'deb http://deb.debian.org/debian jessie-backports main' \
-      > /etc/apt/sources.list.d/jessie-backports.list && \
-    apt-get update -y && \
-    apt-get install -t \
-      jessie-backports \
-      openjdk-8-jdk \
-      ca-certificates-java -y && \
-    # Remove old symlink to set openJDK-8 as default
-    update-alternatives --remove java /usr/lib/jvm/java-7-openjdk-amd64/jre/bin/java
+    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 0xB1998361219BD9C9 && \
+    echo "deb http://repos.azulsystems.com/debian stable main" > /etc/apt/sources.list.d/zulu.list && \
+    apt-get autoremove -y openjdk-11-jre-headless openjdk-11-jdk && \
```

Comment:
why do we have to remove this? is it bundled somewhere in the imported image?

Reply:
Yes, it's included in the image.  I nuked it for a couple reasons: consistent use of the Azul JDKs and to reduce the size of the image.